<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CandyEditor</class>
 <widget class="QMainWindow" name="CandyEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CandyEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btGuardar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSalir">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="GLWidget" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>9</y>
      <width>611</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbCrearMapa">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="title">
     <string notr="true">Crear Mapa</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="tbAncho_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Ancho</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btCrearMapa_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Crear</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbAlto_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Alto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btPersonaje">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>A単adir Personaje</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbPerX">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbPerY">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Y</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbElementos">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Elementos</string>
    </property>
    <widget class="QPushButton" name="btEnemigo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>A単adir Enemigo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btObstaculo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>A単adir Obstaculo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbEneX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbEneY">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbObsY">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbObsX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbParedY">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btPared">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>A単adir Pared</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbParedX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbParedY_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>350</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Y max</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbParedX_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>X max</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbEnemigos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="minimumContentsLength">
      <number>5</number>
     </property>
     <item>
      <property name="text">
       <string>Tipo 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tipo 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tipo 3</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbObstaculos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="minimumContentsLength">
      <number>5</number>
     </property>
     <item>
      <property name="text">
       <string>Tipo 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tipo 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tipo 3</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btSalir</sender>
   <signal>clicked()</signal>
   <receiver>CandyEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
